GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategoryByCospans.gd

```diff
@@ -217,4 +217,54 @@ DeclareAttribute( "GeneralizedMorphismCategoryByCospans",
 DeclareAttribute( "GeneralizedMorphismByCospansObject",
                   IsCapCategoryObject );
 
+####################################
+##
+#! @Section Constructors of lifts of exact functors and natrual (iso)morphisms
+##
+####################################
+
+#! @Description
+#!  AsGeneralizedMorphismByCospan a functor ...
```

Comment:
We should explicitly state that this construction only makes sense for exact functors.